relay-system/Cargo.toml

```diff
@@ -14,8 +14,8 @@ futures = "0.3"
 once_cell = "1.13.1"
 relay-log = { path = "../relay-log" }
 relay-statsd = { path = "../relay-statsd" }
-tokio = { version = "1.0", features = ["macros", "signal"] }
+tokio = { version = "1.0", features = ["macros", "signal", "full"] }
```

Comment:
for some reason, I was unable to run unit tests individually without changing this (compilation error), but i still could do `make test-rust-all` without problem

Reply:
so i'll remove the dependency later when i figure out what the deal is

Reply:
Let's remove it from the diff. If this change exists at this point, the intention is to land it. If you use it for local testing, don't stage it with git.

As for the compilation error you were getting, what is the error?

Reply:
I tried to reproduce this problem locally but was able to run individual tests just fine without the `"full"`, so this must be a local issue. Please remove from diff before merging, as @iker-barriocanal suggested.

Reply:
resolved it on my end now